CRM/Contribute/Form/ContributionPage/Amount.php

```diff
@@ -337,6 +335,7 @@ public static function formRule($fields, $files, $self) {
     if (isset($fields['is_pay_later'])) {
       if (empty($fields['pay_later_text'])) {
         $errors['pay_later_text'] = ts('Please enter the text for the \'pay later\' checkbox displayed on the contribution form.');
+        $this->addElement('checkbox', 'is_monetary', ts('Execute real-time monetary transactions'));
```

Comment:
Perhaps we could remove the checkbox from the form & set it appropriately on postProcess? It would certainly take some digging to be sure that was safe though